### PR TITLE
Billing instructions

### LINKED ISSUE #659: ACH billing for cloud
## Communication goals (e.g. detailed howto vs orientation)

A walkthrough of setting up ACH billing, with screenshots of UI.

### Good locations for content in docs structure

https://docs.citusdata.com/en/v7.3/cloud/support.html

### PR DESCRIPTION
**Pull request by begriffs**
Fixes #659 

The current change is pretty minimal, just some screenshots of an arguably self-explanatory part of the UI. Is there something else we want to explain here?

### CODE REVIEW

**Comment by begriffs** [comment]
@craigkerstiens assigning to you for review since you suggested the topic.

**Comment by sumedhpathak** [comment]
I'll let Craig review it, but visually, the image and the titles blend together. Its hard to distinguish where the CC ends and the ACH section begins. You could break it up by putting some text, but I don't know what text we would add. Is there one method over the other we prefer? Also, are there other keywords we can add so this is searchable?

File: cloud/support.rst [hunk 1]
```diff
@@ -25,6 +28,25 @@ Citus Cloud bills on a per minute basis. We bill for a minimum of one hour of us
...
+
+.. image:: ../images/cloud-bill-menu.png
+
+Then fill out the form for the desired payment option.
+
+Credit Card Option
+~~~~~~~~~~~~~~~~~~
+
+.. image:: ../images/cloud-bill-credit.png
+
```
Comment (sumedhpathak) [inline cloud/support.rst]: Add more space or text to break up these two sections?

**Comment by craigkerstiens** [comment]
Overall good with this, we need to add a bit around the ACH process, feel free to ship this and we can iterate on the other.

**Pull request merged**